evadb/udfs/sentence_feature_extractor.py

```diff
@@ -32,6 +32,7 @@ def setup(self):
             "sentence-transformers/all-MiniLM-L6-v2"
         )
         self.model = AutoModel.from_pretrained("sentence-transformers/all-MiniLM-L6-v2")
+        self.device = None
```

Comment:
Can you remind me why do we need this var?

Reply:
We need to set `self.model_device` -- not sure if `self.device` also needs to be set.

Reply:
On a machine with only a CPU, we get this error -- `has no attribute` `model_device`